core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/SyncTransferProcessManager.java

```diff
@@ -0,0 +1,96 @@
+package org.eclipse.dataspaceconnector.transfer.core.transfer;
+
+import org.eclipse.dataspaceconnector.spi.message.RemoteMessageDispatcherRegistry;
+import org.eclipse.dataspaceconnector.spi.transfer.TransferProcessManager;
+import org.eclipse.dataspaceconnector.spi.transfer.TransferResponse;
+import org.eclipse.dataspaceconnector.spi.transfer.response.ResponseStatus;
+import org.eclipse.dataspaceconnector.spi.transfer.store.TransferProcessStore;
+import org.eclipse.dataspaceconnector.spi.transfer.synchronous.DataProxyManager;
+import org.eclipse.dataspaceconnector.spi.transfer.synchronous.ProxyEntry;
+import org.eclipse.dataspaceconnector.spi.transfer.synchronous.ProxyEntryHandler;
+import org.eclipse.dataspaceconnector.spi.types.TypeManager;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataRequest;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.TransferProcess;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.TransferProcessStates;
+
+import java.net.ConnectException;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import static java.util.UUID.randomUUID;
+import static org.eclipse.dataspaceconnector.spi.types.domain.transfer.TransferProcess.Type.CONSUMER;
+import static org.eclipse.dataspaceconnector.spi.types.domain.transfer.TransferProcess.Type.PROVIDER;
+
+public class SyncTransferProcessManager implements TransferProcessManager {
```

Comment:
I think understood the reason behind the sync/async naming convention, but looking at the implementation, this seems not a sync transfer process manager, more like a ProxyTransferProcessManager.

Reply:
yes and no :)
It is one that allows for proxying, but in that it is synchronous. I'm fine with renaming it though.

Reply:
also, now we use it solely for proxying, but in the future we might have additional use cases.

Reply:
We will find a better name then :)

Reply:
since this PR has been open for quite some time, can we either
- agree on a new name 
- leave this one there for now and find a better one later
and merge it?

Reply:
Also leaving it like it is now and opening an issue with a discussion about renaming is fine for me.

Reply:
ok resolving this now.